### DISCUSSION TITLE
nodegraph - edge colors

### DISCUSSION THREAD

**Discussion by marwin1991** [grafana/grafana#63817] [Dashboards, Panels & Visualizations]
https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/node-graph/

Nodes have following field:

![image](https://user-images.githubusercontent.com/25181517/221668079-05c50656-94d5-4150-acb7-137d8c6090f4.png)

IMO it should be easy and at the same time really useful to have the same field for edges.

![edge_colors](https://user-images.githubusercontent.com/25181517/221669215-c9d08a7b-8d60-4ba7-b80a-4751bf758d35.PNG)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82839. Feel free to continue the discussion around this there. Thank you!
